apps/minifront/src/components/staking/account/delegation-value-view/staking-actions/index.tsx

```diff
@@ -30,17 +41,7 @@ export const StakingActions = ({
    */
   unstakedTokens?: ValueView;
 }) => {
-  const {
-    action,
-    amount,
-    onClickActionButton,
-    delegate,
-    undelegate,
-    onClose,
-    setAmount,
-    validatorInfo: selectedValidatorInfo,
-  } = useStore(stakingSelector);
-
+  const state = useStore(useShallow(stakingActionsSelector));
```

Comment:
There are a number of ways we could handle the syntax here. Using `state`, rather than destructuring the properties, felt cleanest. But we could also do e.g.,:

```suggestion
  const {
    action,
    amount,
    delegate,
    onClickActionButton,
    onClose,
    setAmount,
    undelegate,
    validatorInfo,
  } = useStore(useShallow(stakingActionsSelector));
```

I kind of like using `state`, though, since it makes it very clear in the rest of the component whether a value is coming from props vs. from state.